components/console-backend-service/Dockerfile

```diff
@@ -1,43 +1,39 @@
-FROM golang:1.11.4-alpine3.8 as builder
+FROM eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190930-d28d219 as builder
 
 ENV BASE_APP_DIR /go/src/github.com/kyma-project/kyma/components/console-backend-service
+ENV GO111MODULE on
+
 WORKDIR ${BASE_APP_DIR}
 
-#
-# Copy files
-#
+COPY ./go.mod ./go.sum ./
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
 
-COPY ./vendor/ ${BASE_APP_DIR}/vendor/
-COPY ./internal/ ${BASE_APP_DIR}/internal/
-COPY ./pkg/ ${BASE_APP_DIR}/pkg/
-COPY ./main.go ${BASE_APP_DIR}/
-COPY ./licenses/ ${BASE_APP_DIR}/licenses/
+COPY . .
+
+# Run tests
+RUN make verify
 
-#
 # Build app
-#
+RUN make build
+
+# Copy built app
+RUN mkdir /app
+RUN mv ./console-backend-service /app/main 
 
-RUN CGO_ENABLED=0 GOOS=linux go build -v -o main .
-RUN mkdir /app && mv ./main /app/main && mv ${BASE_APP_DIR}/licenses /app/licenses
+COPY ./licenses/ /app/licenses
 
 FROM alpine:3.10
```

Comment:
You could bump it to 3.11 to solve some security issues (treat it just as a comment, not "request changes")